tla/consensus/ccfraft.tla

```diff
@@ -444,6 +486,30 @@ CommittedTermPrefix(i, x) ==
     \* Otherwise the prefix is the empty tuple
     ELSE << >>
 
+RetirementIndexLog(node_log, i) ==
+    LET 
+        in_indexes = {index \in DOMAIN node_log: 
```

Comment:
The spec is (mostly) camelCase, but unfortunately we don't have a linter/formatter.

Reply:
changed, i think we need a separate pass of the whole spec to sync up on style